src/ui/screens/settings/appPermissions/AppPermissionsList.qml

```diff
@@ -40,7 +40,7 @@ ColumnLayout {
         id: col2
         objectName: "appExclusionsList"
 
-        spacing: MZTheme.theme.windowMargin / 2
+        spacing: MZTheme.theme.vSpacing
```

Comment:
since we are getting rid of the cell's height, I like the extra spacing between rows here, but this also adds extra spacing between the "clear all" button and the first item in the list. 

can we put the list (repeater) inside it's own layout so it can have separate spacing